tiledb/sm/query/readers/reader_base.h

```diff
@@ -199,6 +199,12 @@ class ReaderBase : public StrategyBase {
    */
   std::vector<bool> timestamps_needed_for_deletes_;
 
+  /** Names of dim/attr loaded for query condition. */
+  std::unordered_set<std::string> qc_loaded_attr_names_set_;
+
+  /** Have ve loaded the initial data. */
+  bool initial_data_loaded_;
```

Comment:
Does this need to be guarded for concurrency safety?

Reply:
(also `*v*e` -> `*w*e` above)

Reply:
We don't allow to call query submit on multiple threads. Should be safe unguarded.